documentation/index.html.js

```diff
@@ -811,7 +811,7 @@ Expressions
       object that the current function is attached to. If you pass a function as
       a callback or attach it to a different object, the original value of <code>this</code>
       will be lost. If you're not familiar with this behavior,
-      <a href="http://www.digital-web.com/articles/scope_in_javascript/">this Digital Web article</a>
+      <a href="http://64.13.255.16/articles/scope_in_javascript/">this Digital Web article</a>
```

Comment:
This one kind of bothers me though, the digital-web domain seems to be working correctly. The change was introduced here: https://github.com/jashkenas/coffeescript/commit/f80a8a5120b4b8afd5afecc333486d2306a16303